.env.example

```
@@ -39,3 +39,6 @@ EXTERNAL_API_URL=https://api.example.com
 
 # API 호스트
 API_HOST=api.example.com
+
+# 초대 설정
+INVITATION_EXPIRES_HOURS=48
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)